### PR TITLE
Added initial classification example

### PR DESCRIPTION
**Pull request by jinzhang21**
Note that it only works on local machine yet. I'll add Databricks support later.
The whole notebook works as is. Execution time is about 20s.

Screen Shot 2022-11-10 at 5 31 06 PM

### CODE REVIEW

File: classification/profiles/local.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  model_name: "red_wine_classifier"
+
+# Override the default train / validation / test dataset split ratios
+SPLIT_RATIOS: [0.80, 0.10, 0.10]
+
+INGEST_CONFIG:
+  # For different options please read: https://github.com/mlflow/recipes-classification-template#ingest-step
+  using: csv
+  location: ["./data/winequality-white.csv", "./data/winequality-red.csv"]
+  custom_loader_method: steps.ingest.load_file_as_dataframe
```
Comment (sunishsheth2009) [inline classification/profiles/local.yaml:21]: Change this line to `loader_method: load_file_as_dataframe` to match with the new spec

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,5 @@
 mlflow>=2.0.0rc0
-ipykernel>=6.12.*
-ipython>=7.32.*
+scikit-learn>=1.1
+ipykernel>=6.12
+ipython>=7.32
+flaml
```
Comment (sunishsheth2009) [inline requirements.txt:5]: Do we want to include this in the basic requirement? Specially given that none of the pipelines have an example for flaml? What do you think?

**Review by sunishsheth2009 — APPROVED** [review]

File: requirements.txt (hunk 2 above)
Comment (jinzhang21) [reply requirements.txt:5]: Forgot to remove it. Was testing FLAML. It doesn't work unfortunately.

File: classification/profiles/local.yaml (hunk 1 above)
Comment (jinzhang21) [reply classification/profiles/local.yaml:21]: Good catch!

File: classification/profiles/local.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  model_name: "red_wine_classifier"
+
+# Override the default train / validation / test dataset split ratios
+SPLIT_RATIOS: [0.80, 0.10, 0.10]
+
+INGEST_CONFIG:
+  # For different options please read: https://github.com/mlflow/recipes-classification-template#ingest-step
+  using: csv
+  location: ["./data/winequality-white.csv", "./data/winequality-red.csv"]
+  custom_loader_method: load_file_as_dataframe
```
Comment (sunishsheth2009) [inline classification/profiles/local.yaml:21]: Can we switch `custom_loader_method` to `loader_method`

**Pull request merged**